forge/src/main/java/org/valkyrienskies/mod/forge/mixin/feature/forge_interact/MixinIForgePlayer.java

```diff
@@ -39,6 +40,26 @@ default boolean canReach(final BlockPos pos, final double padding) {
             return true;
         }
     }
+    
+    /**
+     * Include ships in server-side distance check when player interacts with a block.
+     */
+    @Overwrite(remap = false)
+    default boolean canReachRaw(final BlockPos pos, final double padding) {
+        if (VSGameConfig.SERVER.getEnableInteractDistanceChecks()) {
+            double reach = self().getAttributeValue(ForgeMod.BLOCK_REACH.get()) + padding;
+            final Vec3 eyes = this.self().getEyePosition();
+            return VSGameUtilsKt.squaredDistanceBetweenInclShips(this.self().level(),
+                pos.getX() + 0.5,
+                pos.getY() + 0.5,
+                pos.getZ() + 0.5,
+                eyes.x, eyes.y, eyes.z
+            ) < reach * reach;
```

Comment:
```suggestion
            ) <= reach * reach;
```

Forge somehow changed `<` to `<=`, same for above `canReach` but I cannot comment there